### PR TITLE
[Bug Fix] Specifies `daemon off` when running `nginx` command

### PR DESCRIPTION
**Pull request by gruyaume**
# Description

- Specifies daemon off when running nginx command

## Checklist

- [ ] My code follows the [style guidelines](/CONTRIBUTING.md) of this project.
- [ ] I have performed a self-review of my own code.
- [ ] I have made corresponding changes to the documentation.
- [ ] I have added tests that validate the behaviour of the software.
- [ ] I validated that new and existing unit tests pass locally with my changes.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [ ] I have bumped the version of any required library.

### CODE REVIEW

**Review by ghislainbourgeois — APPROVED** [review]
LGTM

**Review by Gmerold — APPROVED** [review]

File: nms-nginx-proxy-operator/src/charm.py [hunk 1]
```diff
@@ -172,18 +172,9 @@ def _configure_pebble(self, event: Union[PebbleReadyEvent, CertificateAvailableE
             )
             self._container.add_layer(self._container_name, self._pebble_layer, combine=True)
             self._container.restart(self._service_name)
-        # TODO: _reload_nginx() is needed by the workaround for not working container.restart()
-        #       and should be removed as soon as the proper Juju mechanism works as expected.
-        self._reload_nginx()
```
Comment (ghislainbourgeois) [inline nms-nginx-proxy-operator/src/charm.py:177]: I assume that you tested that this now works properly with the `daemon off` switch?

**Review by ghislainbourgeois — APPROVED** [review]

File: nms-nginx-proxy-operator/src/charm.py (hunk 1 above)
Comment (gruyaume) [reply nms-nginx-proxy-operator/src/charm.py:177]: Well at this point I'm waiting on integration tests, I'll also test it out manually

Comment (gruyaume) [reply nms-nginx-proxy-operator/src/charm.py:177]: The bundle deploys, the UI shows up and integration tests passed :)

**Pull request merged**